content/millennium/r4/clinical/medications/medication-administration.md

```diff
@@ -48,42 +58,44 @@ The following fields are returned if valued:
- `status`         | N                  | [`token`]     | The [status] of the medication administration event. May be a list separated by commas. Example: `status=completed,not-done`
- `performer`      | N                  | [`reference`] | The identity of the individual who administered the medication. May be a list separated by commas. Example: `performer=1245,7659`
- `effective-time` | N                  | [`date`]      | An date-time range when administration happened. Must include a time. Example: `effective-time=ge2014-05-19T20:54:02.000Z`
- `_count`         | N                  | [`number`]    | The maximum number of results to include in a page. Example: `50`
+ Name             | Required?     | Type          | Description
+------------------|---------------|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ `_id`            | Conditionally | [`token`]     | The logical resource id associated with the resource. It may be a list separated by commas. This parameter is required if the `patient` parameter is not used. Example: `_id=12345`
+ `patient`        | Conditionally | [`reference`] | The specific patient to return medication administrations for. This parameter is required if the `_id` parameter is not used. Example: `patient=1234`
+ `status`         | No            | [`token`]     | The status of the medication administration event. It may be a list separated by commas. Example: `status=completed,not-done`
+ `performer`      | No            | [`reference`] | The identity of the individual who administered the medication. It may be a list separated by commas. Example: `performer=1245,7659`
+ `effective-time` | No            | [`date`]      | A date range for when the administration happened. Example: `effective-time=ge2014-05-19T20:54:02.000Z`
+ `_count`         | No            | [`number`]    | The maximum number of results to include in a page. Example: `50`
 
-Notes:
+_Implementation Notes_
 
-* `effective-time` parameter may be provided 
-  * once with a prefix `ge` or `gt` or `le` or `lt`.   Example: `effective-time=ge2014-05-19T20:54:02.000Z`
-  * twice with the prefixes `ge`, `gt`, `le`, `lt`.    Example: `effective-time=ge2014-05-19T20:54:02.000Z&effective-time=le2014-05-20T20:54:02.000Z`
+- When searching with the `_id` parameter
+  - It must not be provided with any other parameters.
+- When searching with the `effective-time` parameter:
+  - For a single `effective-time` occurrence:
+    - It must be provided with a `ge` `gt` `le` or `lt` prefix to imply the date range to search for administrations within.
+    - The time component is required.
+    - Example: `effective-time=ge2014-05-19T20:54:02.000Z`
+  - For two `effective-time` occurences: 
+    - It must be provided with `le` or `lt` and `ge` or `gt` prefixes to search for administrations within a specific range. 
```

Comment:
nit: 'It' -> 'Both'

Reply:
I misread the logical operators. can we reword a bit to be more explicit that one should be the first and the other the second group?

Reply:
Fixed now